creator-node/src/routes/files.js

```diff
@@ -353,6 +353,54 @@ module.exports = function (app) {
       return errorResponseServerError(e)
     }
 
+    /**
+     * Ensure image files written to disk match dirCID returned from resizeImage
+     */
+
+    const ipfs = req.app.get('ipfsLatestAPI')
+
+    const dirCID = resizeResp.dir.dirCID
+
+    // build ipfs add array
+    let ipfsAddArray = []
+    try {
+      await Promise.all(resizeResp.files.map(async function (file) {
+        const fileBuffer = await fs.readFile(file.storagePath)
+        ipfsAddArray.push({
```

Comment:
does order matter here? or is it just the hash that matters?

Reply:
just confirmed by running this function with this array reversed that it does not matter, good callout tho